src/os/bluestore/BlockDevice.h

```diff
@@ -159,6 +169,17 @@ class BlockDevice {
   virtual bool supported_bdev_label() { return true; }
   virtual bool is_rotational() { return rotational; }
 
+  // HM-SMR-specific calls
+  virtual bool is_smr() { return false; }
```

Comment:
might want to make these getters `const`?

Reply:
Done.

Thanks for the review!